### PR TITLE
feat: add CasdoorUserService and CasdoorAuthService

### PR DESCRIPTION
**Pull request by tangyang9464**
Signed-off-by: tangyang9464 <tangyang9464@163.com>

close: #3 

refer to: [casdoor-go-sdk](https://github.com/casdoor/casdoor-go-sdk)

### CODE REVIEW

File: src/main/java/org/casbin/casdoor/entity/CasdoorUser.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The casbin Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/entity/CasdoorUser.java:1]: 2021

File: src/main/java/org/casbin/casdoor/config/CasdoorConfig.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.casbin.casdoor.config;
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/config/CasdoorConfig.java:1]: license header

File: src/main/java/org/casbin/casdoor/entity/Organization.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The casbin Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/entity/Organization.java:1]: 2021

File: src/main/java/org/casbin/casdoor/util/http/CasdoorResponse.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package org.casbin.casdoor.util.http;
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/util/http/CasdoorResponse.java:1]: header

File: src/main/java/org/casbin/casdoor/util/http/HttpClient.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The casbin Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline src/main/java/org/casbin/casdoor/util/http/HttpClient.java:1]: 2021

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by tangyang9464** [comment]
@hsluoyz changed

**Comment by hsluoyz** [comment]
@fangzhengjin @Kininaru plz review

**Pull request merged**